README.md

```diff
@@ -1,2 +1,32 @@
 # wibrc-eslint-config-vue
 An eslint shareable configuration for Vuejs app
+## 1. Install since github as dev dependency
```

Comment:
Please make a description of this configuration. Read how to write a good and professional `README` file. I think this are going to be good for you for the next projects.